library/cwm/src/lib/cwm/page.rb

```diff
@@ -0,0 +1,56 @@
+require "cwm/custom_widget"
+require "yast"
+Yast.import "CWM"
+
+module CWM
+  # A page widget is a group of widgets that are contained within it.
+  # Several pages are in turn contained within a {Pager}.
+  #
+  # {TreePager} is a {Pager}.
+  #
+  # {Tabs} is a {Pager} and a {Tab} is its {Page}
+  # (FIXME: We haven't made Tabs a subclass of Pager yet.
+  # That will come after we have TreePager working.
+  class Page < CustomWidget
+    # @return [Boolean] is this the initially selected tab
+    attr_accessor :initial
+
+    # @return [WidgetTerm] contents of the tab, can contain {AbstractWidget}s
+    attr_reader :contents
+    # @return [String] Label of {Tab} or of {PagerTreeItem}
+    attr_reader :label
+
+    def initialize(widget_id:, label:, contents:)
```

Comment:
@jreidinger if I understood you correctly a generic Page should do nothing in its constructor and have subclasses override the methods, right?

Only in this class but others too?